.github/workflows/collection.yml

```
@@ -0,0 +1,46 @@
+name: Update collection
+on:
+  issue_comment:
+    types: [created]
+    branches:
+      - gh-pages
+jobs:
+  update_collection:
+    runs-on: ubuntu-latest
+    if: contains(github.event.comment.body, '#sign-off') && (github.event.comment.user.login == 'bamurtaugh' || contains(github.event.comment.user.login, 'devcontainers/maintainers'))
```

```suggestion
    if: contains(github.event.comment.body, '#sign-off') && contains(github.event.comment.user.login, 'devcontainers/maintainers'))
```

Should we add you to devcontainers/maintainers instead?

---

If we can get this check to work, then yes! It seems the check isn't working (as we saw that I needed to add your username explicitly)

---

Resolving since I've added myself (at least temporarily) to the team